mysql-to-kv-delta/function.yaml.values

```diff
@@ -0,0 +1,122 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Container:
+  displayName: Iguazio Data container name
+  kind: string
+  required: true
+  order: 1
+  attributes:
+    defaultValue: bigdata
+
+Table:
+  displayName: Iguazio table name
+  kind: string
+  description: "Name of Iguazio's table destination"
+  required: true
+  order: 2
+  attributes:
+    defaultValue: table
+
+IgzV3f:
+  displayName: V3IO frames endpoint
+  kind: string
+  description: "v3io-frames endpoint including port ID"
+  required: true
+  order: 3
+  attributes:
+    defaultValue: v3io-framesd:8081
+
+IgzUser:
+  displayName: Iguazio user name
+  kind: string
+  required: true
+  order: 4
+  attributes:
+    defaultValue: iguazio
+
+IgzPwd:
+  displayName: Iguazio user password
+  kind: string
+  required: true
+  order: 5
+  attributes:
+    password: true
+
+SqlHost:
+  displayName: MySQL Hostname
+  kind: string
+  required: true
+  order: 6
+
+SqlPort:
+  displayName: MySQL Port ID
+  kind: number
+  required: true
+  order: 7
+  attributes:
+    defaultValue: 3306
+
+SqlDBName:
+  displayName: MySQL DB name
+  kind: string
+  description: "Name of the MySQL database"
+  required: true
+  order: 8
+
+SqlUser:
+  displayName: MySQL Username
+  kind: string
+  required: true
+  order: 9
+
+SqlPwd:
+  displayName: MySQL Password
+  kind: string
+  required: false
+  order: 10
+  attributes:
+    password: true
+
+SqlQuery:
+  displayName: MySQL query
+  kind: string
+  description: "E.G - SELECT * FROM table LIMIT 10"
+  required: true
+  order: 11
+  attributes:
```

Comment:
if there are no attributes then there's no need to put it here (same for the other places it happens)